src/app/components/cron-gen/cron-gen.jsx

```diff
@@ -25,9 +26,10 @@ const useStyles = makeStyles({
   datetimeLayout: {
     display: 'flex',
     flexDirection: 'column',
+    gap: '8px',
+    padding: '16px 0px',
     [theme.breakpoints.up('md')]: {
       justifyContent: 'space-between',
```

Comment:
Is this class still needed? If not, maybe we can delete this entire breakpoints thing